src/nvim/drawline.c

```diff
@@ -1021,21 +1021,25 @@ static void win_line_continue(winlinevars_T *wlv)
   }
 }
 
-/// Display line "lnum" of window 'wp' on the screen.
+/// Display line "lnum" of window "wp" on the screen.
+/// Start at row "startrow", stop when "endrow" is reached.
+/// When "number_only" is true only update the number column.
+/// "spv" is used to store information for spell checking, kept between
+/// sequential calls for the same window.
```

Comment:
Is this desired? I noticed it was missing but I guess it is included in short-form in the @param descriptions.

Reply:
What was missing?

Reply:
(Everything added here?) I mean I noticed that the function comment for our `win_line()` was much shorter. But I guess the information is included in the param comments instead.
Do we want both or should I remove it again?

Reply:
I think they should be moved to the param comments.